cmake/VerifyCompiler.cmake

```diff
@@ -35,11 +35,11 @@ elseif(HIP_PLATFORM STREQUAL "hcc")
     else()
         # Workaround until hcc & hip cmake modules fixes symlink logic in their config files.
         # (Thanks to rocBLAS devs for finding workaround for this problem.)
-        list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hcc /opt/rocm/hip)
-        # Ignore hcc warning: argument unused during compilation: '-isystem /opt/rocm/hip/include'
+        list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX}/hcc ${CMAKE_INSTALL_PREFIX}/hip)
```

Comment:
Same comment as for rocPRIM:
Does this impact the build and install documentation for this component?

What happens when this component package is built without CMAKE_INSTALL_PREFIX and installed in a multi ROCm installed system?

Reply:
CMAKE_INSTALL_PREFIX is default set to /opt/rocm, so if a user want to build the rocRAND to a new path, then CMAKE_INSTALL_PREFIX needs to be set.
Regarding building when multiple ROCm version are installed, need to set /opt/rocm link to the required version and CMAKE_INSTALL_PREFIX can be set/unset, accordingly the rocRAND can be built.